test/parallel/test-inspector-workers-flat-list.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+
+const { Session } = require('inspector');
+const { Worker, isMainThread, parentPort } = require('worker_threads');
+
+const MAX_DEPTH = 3;
+
+let rootWorker = null;
+
+function runTest() {
+  let reportedWorkersCount = 0;
+  const session = new Session();
+  session.connect();
+  session.on('NodeWorker.attachedToWorker', ({ params: { workerInfo } }) => {
+    console.log(`Worker ${workerInfo.title} was reported`);
+    if (++reportedWorkersCount === MAX_DEPTH) {
+      rootWorker.postMessage({ done: true });
+    }
+  });
+  session.post('NodeWorker.enable', { waitForDebuggerOnStart: false });
+}
+
+function processMessage({ child }) {
+  console.log(`Worker ${child} is running`);
+  if (child === MAX_DEPTH) {
+    runTest();
+  }
+}
+
+function workerCallback(message) {
+  parentPort.postMessage(message);
+}
+
+function startWorker(depth, messageCallback) {
+  const worker = new Worker(__filename);
+  worker.on('message', messageCallback);
+  worker.postMessage({ depth });
+  return worker;
+}
+
+function runMainThread() {
+  rootWorker = startWorker(1, processMessage);
+}
+
+function runWorkerThread() {
+  let worker = null;
```

Comment:
Can you rename this to `child` or `childWorker` to be more explicit?

Reply:
Done